pkgs/by-name/qb/qbittorrent-exporter/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGo125Module,
+  nix-update-script,
+}:
+buildGo125Module rec {
```

```suggestion
buildGo125Module (finalAttrs: {
```
`finalAttrs` is a better pattern than `rec`: https://github.com/NixOS/nixpkgs/issues/315337